crates/primitives/src/hardfork.rs

```diff
@@ -73,31 +73,33 @@ pub mod name {
     pub const LATEST: &str = "Latest";
 }
 
-impl From<&str> for SpecId {
-    fn from(name: &str) -> Self {
+impl TryFrom<&str> for SpecId {
+    type Error = ();
```

Comment:
I think this is just `FromStr` now

can we introduce an error here because otherwise this can't be used with `.parse()?` for example

Reply:
Addressed by https://github.com/bluealloy/revm/pull/2236/commits/a866ac05c7c59a110b9bd2707ff0762220129979